modules/EvseManager/evse/evse_managerImpl.cpp

```diff
@@ -181,9 +168,7 @@ void evse_managerImpl::ready() {
             }
 
             session_started.reason = start_reason;
-
-            set_session_uuid();
-
+            const auto session_uuid = this->mod->charger->get_session_id();
```

Comment:
previously set_session would always ensure that we have a uuid - here we only get it, which relies that we call `startSession` before - is this somehow ensured?

Reply:
This code is executed as part of the `start_session` function so its guaranteed that the session_uuid is set here